components/cam/cime_config/config_component.xml

```diff
@@ -96,6 +96,38 @@
       <value compset="_CAM[45]%WCBC"  >-carma bc_strat</value>
       <value compset="_CAM4%WCSF"     >-chem waccm_mozart_sulfur -carma sulfate</value>
       <value compset="_CAM[45]%WCMX"  >-waccmx</value>
+      <!-- old MMF / super-parameterization compsets -->
```

Comment:
NOTE: these are in here right now only to maintain testing against the ECP fork. These will go away in a future PR.

Reply:
I'll add a note in the file